prelude/bootstrap.js

```diff
@@ -2029,6 +2029,66 @@ function payloadFileSync(pointer) {
     }
   }
 
+  function wrappedSpawnFunc(fn) {
+    return function wrappedSpawn(options) {
+      function cleanup(tmp) {
+        if (!fs.existsSync(tmp)) return;
+        const major = parseInt(process.versions.node.split('.')[0], 10);
+        if (major >= 16) {
+          fs.rmSync(tmp, { recursive: true });
+        } else if (major > 12) {
+          fs.rmdirSync(tmp, { recursive: true });
+        } else {
+          const files = fs.readdirSync(tmp);
+          for (const file of files) {
+            fs.unlinkSync(path.join(tmp, file));
+          }
+        }
+      }
+
+      function shouldRunUsingCmd(file) {
+        return (
+          process.platform === 'win32' &&
+          (file.endsWith('.cmd') || file.endsWith('.bat'))
+        );
+      }
```

Comment:
Is this really necessary? 

Shouldn't the users do this themselves?

Reply:
Without it it won't be running on windows, I think it's an even worse idea to search for a path in the args.